pwiz_tools/Skyline/SkylineTester/SkylineTesterWindow.cs

```diff
@@ -1939,6 +1939,11 @@ private void buttonRunStatsExportCSV_Click(object sender, EventArgs e)
             _tabRunStats.ExportCSV();
         }
 
+        private void runSerial_CheckedChanged(object sender, EventArgs e)
+        {
+            Offscreen.Enabled = runSerial.Checked; // Everything happens offscreen in parallel tests, so don't offer the option

```

Comment:
Could we make this disabled and checked to make it clear that you don't have an option, but the only option is offscreen?

Reply:
No problem.